src/components/__tests__/action-button.spec.js

```diff
@@ -159,12 +159,12 @@ describe("action button", () => {
   });
   describe("runActions", () => {
```

Comment:
Shouldn't there be a new test, so we have one with the prop off and one with it on?

Reply:
Yes